packages/node/src/models/app-instance.ts

```diff
@@ -65,6 +65,7 @@ export class AppInstance {
     latestState: any,
     latestVersionNumber: number,
     latestTimeout: number,
+    outcomeType: OutcomeType,
```

Comment:
Currently this is implied by which of `twoPartyOutcomeInterpreterParams` or `coinTransferInterpreterParams` are `undefined`. Is this field necessary?

Reply:
I was completing the TODO you wrote at https://github.com/counterfactual/monorepo/blame/88d8b347adce9c150550f467f64e57641f303009/packages/node/src/protocol/utils/get-outcome-increments.ts#L72